### PR TITLE
Add example of setting the data of XML nodes

### PR DESCRIPTION
**Pull request by pyatnitsev**
Add an example serialization of node with attribute and value in same time

### CODE REVIEW

File: components/serializer.rst [hunk 1]
```diff
@@ -885,6 +885,18 @@ The array keys beginning with ``@`` are considered XML attributes::
     //     <foo bar="value" />
     // </response>
 
+If you want create a node with value and attribute use this::
+
+    array('foo' => array('@bar' => 'value', # => 'baz'));
```
Comment (javiereguiluz) [inline components/serializer.rst]: minor syntax issue: `# => 'baz'` should be `'#' => 'baz'`

Comment (pyatnitsev) [reply components/serializer.rst]: agree with you, thanks

**Comment by javiereguiluz** [comment]
@pyatnitsev thanks for this contribution. I've slightly reworded the content because XML terminology is sometimes tricky (at least to me it is 😊). "value" is usually reserved to "attribute values", so it was a bit confusing to say "attributes and values" ... we can say "attributes and data":

xml-terms

**Comment by pyatnitsev** [comment]
@javiereguiluz, thank you, sorry for incorrect branch.

**Pull request closed without merging**